modules/swagger-codegen/src/main/resources/python/model.mustache

```diff
@@ -30,6 +35,11 @@ class {{classname}}:
             {{/vars}}{{newline}}
         }
 
+        self.attributeMap = {
```

Comment:
The instance variable `attributeMap` is `dict`, which key is the attribute name and value is the json key in model definition.